Stylesheets/product.css

```diff
@@ -0,0 +1,63 @@
+body{
+    background-color: var(--st-white);
+}
+
+.little-margin-bottom{
+    margin-bottom: 0.5rem;
+}
+
+.product-status{
+    display: inline-block;
+   margin: 0 1rem 2rem 0.5rem;
+}
+
+.flex-spacebetween{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* card wishlist */
+.wishlist-badge{
+    width: 25px;
+    height: 23px;
```

Comment:
use rems here instead of px.....why we do that is to make sure that if the screen is small it wouldn't look really small/unnoticable and in small screens it wouldn't be larger than needed..

Reply:
the badge when provided with the relative units proved to be unstable that is why i used px